server/app/views/components/ButtonStyles.java

```diff
@@ -64,12 +63,12 @@ public final class ButtonStyles {
       StyleUtils.joinStyles(BUTTON_BASE_SOLID_BLUE_SEMIBOLD, "text-xl");
 
   public static final String OUTLINED_TRANSPARENT =
-      StyleUtils.joinStyles(BUTTON_BASE_OUTLINE_SEMIBOLD, "text-base");
+      StyleUtils.joinStyles(BUTTON_BASE_OUTLINE_SEMIBOLD, "text-base", "bg-transparent");
```

Comment:
Why: The new ordering in the css file causes `bg-transparent` to load later now upsetting the visuals. Unlike the other `bg-transparent` changes removing the class, this one is adding it for the same general reason.